docs/publishing-binaries.md

```diff
@@ -157,10 +157,25 @@ index e00b7b5..22f7cd9 100644
 
 3. Publishing C++ binaries by pushing a commit with `[publish binary]` per https://github.com/mapbox/node-cpp-skel/blob/master/docs/publishing-binaries.md#7-all-done
 
-4. Require your module in downstream applications like:
+
+4. 
```

Comment:
To fix this formatting, I think you can use the "three spaces as a line break" trick:

Example markdown (note the highlighted _three spaces_ at the end of the option A line):

<img width="407" alt="screen shot 2018-04-06 at 10 27 55 am" src="https://user-images.githubusercontent.com/1943001/38435015-42b4f470-3985-11e8-9b4a-81bb63a94bb5.png">

Example output: 

<img width="385" alt="screen shot 2018-04-06 at 10 28 03 am" src="https://user-images.githubusercontent.com/1943001/38435025-4803ca46-3985-11e8-95fb-1f170a77fe0b.png">

Reply:
done!